modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -365,3 +365,56 @@ The Logging exporter prints data to the standard output.
           exporters: [logging]
 ----
 <1> Verbosity of the logging export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+
+[id="extensions_{context}"]
+=== Extensions
+
+[id="basicauth-extension_{context}"]
+==== BASICAUTH extension
+
+The BasicAuth extension can be used as an authenticator on receivers and exporters.
+The configuration must specify only one instance of the BasicAuth extension: for client authentication or server authentication.
```

Comment:
```suggestion
Only one instance of the BasicAuth extension is supported in the OpenTelemetry Collector custom resource. Client authentication and server authentication are configured in separate sections in the OpenTelemetry Collector custom resource.
```

Reply:
> Only one instance of the BasicAuth extension is supported in the OpenTelemetry Collector custom resource

What do you mean by one instance?

Reply:
I mean this: https://github.com/max-cx/openshift-docs/pull/13/commits/b7f9b8bdb88bb77ad121b7427a037c3daf1d934f#r1334068446.

Reply:
I mean this: https://github.com/max-cx/openshift-docs/pull/13/commits/b7f9b8bdb88bb77ad121b7427a037c3daf1d934f#r1334068446

Reply:
Looking at your original sentence:
`The configuration should specify only one instance of basicauth extension for either client or server authentication.`
Do we need this part at all: `only one instance of basicauth extension`?

Reply:
How about this?
```suggestion
Client authentication and server authentication for the BasicAuth extension are configured in separate sections in the OpenTelemetry Collector custom resource.
```